app/main.py

```diff
@@ -2,0 +1,38 @@
+class Animal:
+    def __init__(self, name: str, appetite: int, is_hungry: bool = 1) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
```

Comment:
Consider using `True` instead of `1` for the `is_hungry` attribute to maintain consistency with boolean data types.